app/routes/events/components/EventDetail/index.js

```diff
@@ -236,7 +240,7 @@ export default class EventDetail extends Component<Props> {
               <Flex column>
                 <h3>Påmeldte</h3>
                 {registrations ? (
-                  [
+                  registrations.sort(sortRegistrationsBySharedMemberships) && [
```

Comment:
This depends on `.sort()` being in-place, which it is documented as being [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort).

Reply:
Imo this should be done inside the selector, not here.

It is also easier to use lodash `sortBy`, something like:
```jsx
sortBy(registrations, 'sharedMemberships')
```